tracer/src/Datadog.Trace/Iast/IastRequestContext.cs

```diff
@@ -39,6 +42,16 @@ internal void AddIastVulnerabilitiesToSpan(Span span)
         {
             span.Tags.SetTag(Tags.IastJson, _vulnerabilityBatch.ToString());
         }
+
+        if (_executedTelemetryHelper != null)
+        {
+            var tags = new List<Tuple<string, int>>();
+            _executedTelemetryHelper.GenerateMetricTags(tags);
+            foreach (var tag in tags)
+            {
+                span.Tags.SetMetric(tag.Item1, tag.Item2);
+            }
+        }
```

Comment:
This wasn't quite what I meant, I meant this:

```suggestion
        if (_executedTelemetryHelper != null)
        {
            _executedTelemetryHelper.GenerateMetricTags(span.Tags);
        }
```

(changes to `GenerateMetricTags()` shown below!)